### PR TITLE
Added form to projects page

### LINKED ISSUE #16: Project Form - Projects Page

### PR DESCRIPTION
**Pull request by NathanBlackman**
Added to the projectsFormBuilder and rendered it to the dom

### CODE REVIEW

**Review by austincparker — DISMISSED** [review]
goal!

File: scripts/utils/render.js [hunk 1]
```diff
@@ -100,7 +103,17 @@ const packageFormBuilder = () => {
 };
 
 const projectsFormBuilder = () => {
-  return;
+    const domString = `
+        <form>
+            <label for="board-name">Project board name</label><br>
```
Comment (pmpurcell) [inline scripts/utils/render.js]: Change ids at 109, 111, and 112 to camelCase (ie "boardName")

Otherwise, it all looks good!

**Review by pmpurcell — DISMISSED** [review]
Looks good!

**Review by austincparker — APPROVED** [review]
goal!

**Review by pmpurcell — APPROVED** [review]
LGTM!

**Pull request merged**